venues/ICML.cc/2017/MLAV/python/submission_done.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python
```

Comment:
Here is another case: https://github.com/iesl/openreview-scripts/blob/master/venues/swsa.semanticweb.org/ISWC/2017/DeSemWeb/python/create-invitations.py